packages/extractor/source/lib/package-helpers.ts

```diff
@@ -118,16 +124,31 @@ export async function downloadPackage({
   const tarballUrl = await getPackageVersionTarballUrl(name, version)
   await downloadPackageTarball(tarballUrl, tarballDownloadDir)
   await decompress(tarballDownloadDir, tarballDecompressDir)
-  fs.move(path.join(tarballDecompressDir, 'package'), downloadDir, { overwrite: true })
```

Comment:
This was changed as sometimes the decompress package does not have a folder called 'package' but for example '@types/react' returns a folder named 'react'